Server/GameEngine.cpp

```diff
@@ -26,6 +26,12 @@ void GameEngine::removeGameObjectById(int id) {
 	}
 }
 
+void GameEngine::init() {
+	gameState.in_progress = false;
+	gameState.score = std::make_tuple(1, 2);
```

Comment:
Default score should be 0,0 right? I'm assuming this current value is just for testing